frontend/src/index.html

```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 <html>
   <head>
+    <link rel="stylesheet" href="node_modules/react-table/react-table.css">
```

Comment:
This won't work, as webpack can't resolve the `link` to include `react-table` in the bundle. Rather import it in `index.tsx` when necessary.

Reply:
It did, but still get what you're saying